src/besselj.jl

```diff
@@ -150,6 +150,29 @@ function besselj1(x::Float32)
         return p * s
     end
 end
+"""
+    besselj(nu, x::T) where T <: Union{Float32, Float64}
+
+Bessel function of the first kind of order nu, ``J_{nu}(x)``.
+Nu must be real.
+"""
+function _besselj(nu, x)
+    nu == 0 && return besselj0(x)
+    nu == 1 && return besselj1(x)
+    if x < 30.0
+        if nu > 60.0
+            return log_besselj_small_arguments_orders(nu, x)
+        else
+            return besselj_small_arguments_orders(nu, x)
+        end
+    elseif 2*x < nu
+        return besselj_debye(nu, x)
+    elseif x > 2*nu
```

Comment:
isn't this just an `else`?

Reply:
There's really going to be 4 algorithms that will come into play.
1. Asymptotic expansion for x > 5nu
2. Power series for small arguments
3. Debye expansion for x < nu
4. Fall back routine for when x~nu that is undetermined.

The else branch will be the 4th algorithm just haven't gotten to that yet.